peekingduck/pipeline/nodes/input/live.py

```diff
@@ -24,7 +24,47 @@
 
 
 class Node(AbstractNode):
-    """Node to receive livestream as inputs."""
+    """Node to receive livestream as inputs.
+
+    Inputs:
+        None
+
+    Outputs:
+        |img|
+
+        |filename|
+
+        |pipeline_end|
+
+        |saved_video_fps|
+
+    Configs:
+        fps_saved_output_video (:obj:`int`): **default = 10**
+
+            FPS of the mp4 file after livestream is processed and exported.
+            FPS dependant on running machine performance.
+
+        filename (:obj:`str`):  **default = "webcamfeed.mp4"**
+
+            Filename of the mp4 file if livestream is exported.
+
+        resize (:obj:`Dict`): **default = { do_resizing: False, width: 1280, height: 720 }**
+
+            Dimension of extracted image frame
+
+        input_source (:obj:`int`): **default = 0 (for webcam)**
+
+            Refer to OpenCV doucmentation for list of source stream codes
```

Comment:
link to the relevant page in OpenCV using 
```suggestion
            Refer to `OpenCV doucmentation <https//www.google.com>`_ for list of source stream codes
```

Reply:
Thanks. I have made the changes